data-in-pipeline/infra/__main__.py

```diff
@@ -160,7 +160,8 @@ def generate_secret_key(project: str, aws_service: str, name: str):
 
 
 aurora_cluster = aws.rds.Cluster(
-    f"{name}-aurora-cluster",
+    f"{name}-{environment}-aurora-cluster",
+    cluster_identifier=f"{name}-{environment}-aurora-cluster",
```

Comment:
I am always a little wary of putting the `environment` into the name? 

Reason being is that the environment is forked right at the top of the chain i.e. the AWS account, so not sure what value it adds and means you need to then use this pattern downstream making all apps etc environment aware.

For example, the AppRunner instance could just connect to `data-in-pipeline-aurora-cluster` without needing any config.

As soon as this differs from env to env, we then need to pass that value down and make the service configurable, with not much benefit.